JS/models/ExperimentalTypes.js

```diff
@@ -1,8 +1,8 @@
 const ExperimentalTypes   = {
-    LIFTING_BODY: 'lifting_body',
-    HYPERSONIC: 'HYPERSONIC',
-    HIGH_ALTITUDE: 'HIGH_ALTITUDE',
+    LIFTING_BODY: 'liftingBody',
+    HYPERSONIC: 'hypersonic',
+    HIGH_ALTITUDE: 'highAltitude',
     VTOL: "VTOL"
```

Comment:
Usually, we use CAPITAL_LETTERS notation for constants. Please correct.
Also, please give more details about this `VTOL` Type. It is not clear for people not familiar with the domain.

Reply:
Done